lib/sales_analyst.rb

```diff
@@ -0,0 +1,80 @@
+class SalesAnalyst
+  include Math
+
+  attr_reader :engine
```

Comment:
do we need an atter_reader for the engine? i don't think we do, because we are only using it to talk "out" to the sales engine, the sales engine can talk directly with sales analyst with out it.